README.rst

```diff
@@ -161,20 +160,29 @@ The tests are also run automatically with `Travis CI <https://travis-ci.org/RaRe
 Supported Compression Formats
 -----------------------------
 
-``smart_open`` allows reading and writing gzip, bzip2 and xz files.
+``smart_open`` allows reading and writing gzip and bzip2 files.
 They are transparently handled over HTTP, S3, and other protocols, too, based on the extension of the file being opened.
-You can easily add support for other file extensions and compression formats:
+You can easily add support for other file extensions and compression formats.
+For example, to open xz-compressed files:
 
 .. code-block:: python
 
-    def _handle_lzma(file_obj, mode):
-        import lzma
-        return lzma.LZMAFile(filename=file_obj, mode=mode, format=lzma.FORMAT_ALONE)
-
+    import lzma, os
```

Comment:
Can you make sure that [doctest](https://docs.python.org/2/library/doctest.html) picks up this example? Have a look at the above examples: stuff that starts with ">>>" and "..." are commands that get run through the Python interpreter. You want all lines in your example to start like that.